### DISCUSSION TITLE
[Questions] [No details to work with] quorum queue consumer cannot be recovered

### DISCUSSION THREAD

**Discussion by dormanze** [rabbitmq/rabbitmq-server#12660] [Questions] [answered]
### Community Support Policy

- [X] I have read [RabbitMQ's Community Support Policy](https://github.com/rabbitmq/rabbitmq-server/blob/main/COMMUNITY_SUPPORT.md)
- [X] I agree to provide all relevant information (versions, logs, rabbitmq-diagnostics output, detailed reproduction steps)

### RabbitMQ version used

3.13.6

### Erlang version used

26.2.x

### Operating system (distribution) used

linux

### How is RabbitMQ deployed?

Generic binary package

### rabbitmq-diagnostics status output

See https://www.rabbitmq.com/docs/cli to learn how to use rabbitmq-diagnostics
<details>

### Logs from node 1 (with sensitive values edited out)

See https://www.rabbitmq.com/docs/logging to learn how to collect logs
<details>

```
# PASTE LOG HERE, BETWEEN BACKTICKS
```
</details>

### Logs from node 2 (if applicable, with sensitive values edited out)

See https://www.rabbitmq.com/docs/logging to learn how to collect logs
<details>

```
# PASTE LOG HERE, BETWEEN BACKTICKS
```
</details>

### Logs from node 3 (if applicable, with sensitive values edited out)

See https://www.rabbitmq.com/docs/logging to learn how to collect logs
<details>

```
# PASTE LOG HERE, BETWEEN BACKTICKS
```
</details>

### rabbitmq.conf

See https://www.rabbitmq.com/docs/configure#config-location to learn how to find rabbitmq.conf file location
<details>

```
# PASTE rabbitmq.conf HERE, BETWEEN BACKTICKS
```
</details>

### Steps to deploy RabbitMQ cluster

RabbitMQ cluster with three nodes.

### Steps to reproduce the behavior in question

After the data disk permission of one RabbitMQ is changed to 000, RabbitMQ crashes and consumers in the arbitration queue do not automatically recover.
chmod 000 -R /opt/rabbitmq/data

### advanced.config

See https://www.rabbitmq.com/docs/configure#config-location to learn how to find advanced.config file location
<details>

```
# PASTE advanced.config HERE, BETWEEN BACKTICKS
```
</details>

### Application code

<details>

```python
# PASTE CODE HERE, BETWEEN BACKTICKS
```
</details>

### Kubernetes deployment file

<details>

```yaml
# Relevant parts of K8S deployment that demonstrate how RabbitMQ is deployed
# PASTE YAML HERE, BETWEEN BACKTICKS
```
</details>

### What problem are you trying to solve?

We use the Java client, where the automatic recovery of the client is disabled, and we actively listen to the connection and channel shutdown events. When an exception is triggered, the server crashes occasionally and does not notify the client of the shutdown event. As a result, the client does not recover automatically. I added some logs to the server, and the consumer was always deleted by a strange channel, which was not the channel that created the consumer.

**Comment by michaelklishin** [comment, marked as answer]
@dormanze look at you, first ticked the box saying that you will provide all relevant information and ended up providing next to nothing relevant. Do you think we are asking for those logs for the kicks of it? Should on top of that we guess what does "server crashes" means, e.g. what exception the node logs?

In any case, 3.13 is [out of community support](https://www.rabbitmq.com/release-information). Try against against `4.0.3` and provide all logs and more detailed descriptions this time around.
